docs/features/DefaultInterfaceImplementation.md

```diff
@@ -27,6 +27,41 @@ class Test1 : I1
 }
 ```
 
+- Re-abstraction of interface implementation in derived interfaces. Types implementing the derived interface are required to supply implementation even when a base interface provides an implementation. 
+Here is an example:
+```
+public interface I1
+{
+    void M1() 
+    {
+    }
+
+    int P1 
+    {
+        get => throw null;
+        set => throw null;
+    }
+
+    event System.Action E1
+    {
+        add => throw null;
+        remove => throw null;
+    }
+}
+
+public interface I2 : I1
+{
+    abstract void I1.M1();
```

Comment:
>abstract [](start = 4, length = 8)

In an interface, when a body is missing `abstract` is implied.  So presumably the `abstract` keyword is optional here.  However I see no tests for that.  Please add an example and/or comment here and tests covering this.

Reply:
> In an interface, when a body is missing abstract is implied.  

It is not implied for interface implementations and I am not sure if it is a good idea to do that. The behavior is captured in MethodImplementationInDerived_04 and similar unit-tests. I am not planning to change that in the current PR, instead I'll bring that for discussion to LDM. 

---
In reply to: [285270178](https://github.com/dotnet/roslyn/pull/35756#discussion_r285270178) [](ancestors = 285270178)

Reply:
Added to [https://github.com/dotnet/csharplang/blob/master/meetings/2019/README.md](https://github.com/dotnet/csharplang/blob/master/meetings/2019/README.md) under "Schedule ASAP"

---
In reply to: [285274070](https://github.com/dotnet/roslyn/pull/35756#discussion_r285274070) [](ancestors = 285274070,285270178)